bio/reference/inphared-db/wrapper.py

```diff
@@ -0,0 +1,9 @@
+__author__ = "Noriko A. Cassman"
+__copyright__ = "Copyright 2023, Noriko A. Cassman"
+__email__ = "noriko.cassman@gmail.com"
+__license__ = "MIT"
+
+from snakemake.shell import shell
+
+    shell:
+        "curl {params.prefix}{params.date}{params.suffix} -o {params.date}{params.suffix}"
```

Comment:
Some little things here:
1. You have to reference anything from the snakemake rule via the `snakemake` dict. So for example `snakemake.params.date`.
2. We should only use the full `output` file name here, so that users can put in there whatever they want.
3. We have to call the `shell()` function here, as this is not the body of an actual rule, but rather a plain python script.
4. We have to use the `f""` construct here to use format strings to fill in the variables in this context. Here, we are not dealing with snakemake wildcards, but rather python variables. The `{}` syntax is the same, so this is very confusing...
5. I would manually put in the separator between URL and file name here, just for a slightly clearer structure of the download link. Then, we can remove the trailing`/` in the url in `config.yaml`.

```suggestion
    shell(f"curl {snakemake.params.url}/{snakemake.params.date}{snakemake.params.suffix} -o {snakemake.output}"
```